charts/deploy/templates/grafana-oauth-session-secret.yaml

```diff
@@ -5,4 +5,4 @@ data:
 kind: Secret
```

Comment:
@eformat I think this template is no longer used anyway. You can just delete it.

Reply:
@etsauer - no, looks like its still used here - https://github.com/redhat-cop/pelorus/blob/master/charts/deploy/templates/grafana.yaml#L61